configure.py

```diff
@@ -43,7 +43,14 @@
 _ROOTDIR = op.abspath(op.dirname(__file__))
 _SYSTEM = "MinGW" if sysconfig.get_platform().startswith("mingw") else platform.system()
 _RENDERDOC_ID = f"renderdoc_{_SYSTEM}"
+_FFMPEG_ID = f"ffmpeg_{_SYSTEM}"
```

Comment:
I don't think we will need FFmpeg manually pulled anywhere else; you can probably just use `ffmpeg` identifier, just like egl/opengl registry and pkgconf.

Reply:
Done.